### DISCUSSION TITLE
Possible to add sonner (toast notif)?

### DISCUSSION THREAD

**Discussion by jaeyson** [fluxonui/fluxon#13] [Feature Requests]
Hi! is it possible to add [sonner - a toast notif](https://sonner.emilkowal.ski)?

**Comment by andrielfn** [comment]
Hey @jaeyson, I wasn't aware of Sonner before, but I really like it! It looks beautifully designed. 🎨

I've considered creating a Toast component, but I haven't delved into it yet. I would probably aim for something similar to Sooner—well designed, nice animations, customizable options, support for multiple messages, auto-closing, etc. I believe the most challenging part would be ensuring compatibility with the Phoenix flash messages, as I think that's what people would expect. I’ll need to explore this further to gain a better understanding.

In any case, I’m definitely want to explore a Toast component in the near future. Right now I'm working on the date(time) picker and an autocomplete that are still in progress.

**Reply by NduatiK** [reply]
@jaeyson, LiveToast will probably meet your needs: 
https://toast.src.rip/

**Reply by jaeyson** [reply]
@NduatiK yes, for now. LiveToast is a great package. But I asked here in FluxonUI for a feature request.

**Comment by tlvenn** [comment]
For completeness there is also [Flashy](https://github.com/sezaru/flashy) which supports rendering [custom notifications](https://github.com/sezaru/flashy?tab=readme-ov-file#adding-a-entirely-custom-notification)
